### PR TITLE
Reduced digitalobjects list

### PR DESCRIPTION
**Pull request by clorenz**
Return a list of digitalobjects only with their metadata (all identifiers and last modification date)

### CODE REVIEW

File: dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/EntityRepositoryImpl.java [hunk 1]
```diff
@@ -325,6 +332,65 @@ public E findOneByRefId(long refId) {
     return (E) result.orElse(null);
   }
 
+  @Override
+  public List<E> findAllMetadata(EntityType entityType) {
```
Comment (datazuul) [inline dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/EntityRepositoryImpl.java:336]: umbenennen nach findAllReduced

File: dc-cudami-server/dc-cudami-server-business/src/main/java/de/digitalcollections/cudami/server/business/api/service/identifiable/entity/DigitalObjectService.java [hunk 2]
```diff
@@ -29,4 +29,6 @@
       DigitalObject digitalObject, List<FileResource> fileResources);
 
   List<FileResource> saveFileResources(UUID digitalObjectUuid, List<FileResource> fileResources);
+
+  List<DigitalObject> findAllMetadata();
```
Comment (datazuul) [inline dc-cudami-server/dc-cudami-server-business/src/main/java/de/digitalcollections/cudami/server/business/api/service/identifiable/entity/DigitalObjectService.java:33]: s. oben

File: dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/EntityRepositoryImpl.java (hunk 1 above)
Comment (clorenz) [reply dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/EntityRepositoryImpl.java]: done

File: dc-cudami-server/dc-cudami-server-business/src/main/java/de/digitalcollections/cudami/server/business/api/service/identifiable/entity/DigitalObjectService.java (hunk 2 above)
Comment (clorenz) [reply dc-cudami-server/dc-cudami-server-business/src/main/java/de/digitalcollections/cudami/server/business/api/service/identifiable/entity/DigitalObjectService.java]: ebenso :)

**Review by datazuul — APPROVED** [review]

**Pull request merged**